srcpkgs/runit-void/template

```diff
@@ -8,7 +8,7 @@ homepage="http://www.voidlinux.eu"
 short_desc="Void Linux runit scripts"
```

Comment:
shouldn't homepage now be https://www.voidlinux.org ?

Reply:
No, homepage should be for the project itself, not necessarily the project it belongs to.